src/validator.spec.ts

```diff
@@ -1,82 +1,220 @@
-import { validator, setErrorMessage } from './validator';
+import { setErrorMessage, validator } from './validator';
 
-// TODO: Add specs
 describe('fonk-previous-date-validator specs', () => {
-  it('should return succeeded validation when it feeds value equals undefined', () => {
-    // Arrange
-    const value = void 0;
+  describe('Date option boundaries', () => {
+    it('should return succeeded validation when value is a valid Date object earlier than actual Date', () => {
+      const value = new Date(2018, 11, 24, 10, 33, 30, 0);
+      const date = new Date();
+
+      const result = validator({ value, customArgs: { date } });
+
+      expect(result).toEqual({
+        succeeded: true,
+        message: '',
+        type: 'PREVIOUS_DATE',
+      });
+    });
+
+    it('should return succeeded validation when value is a valid Date object earlier than customArgs date param', () => {
+      const value = new Date(2018, 11, 24, 10, 33, 30, 0);
+      const date = new Date(2018, 12, 30, 15, 33, 30, 0);
```

Comment:
Be careful with this date. This date isn't `December 2018`

Reply:
Replaced all Dates with 12 as month => f666b0294aa1bbded5fe45811d079f519857226f